Spigot-API-Patches/0026-Add-command-to-reload-permissions.yml.patch

```diff
@@ -0,0 +1,99 @@
+From 432793805fe1751b4ac8c5c7d943531abb7f1d56 Mon Sep 17 00:00:00 2001
+From: William <admin@domnian.com>
+Date: Fri, 18 Mar 2016 03:28:07 -0400
+Subject: [PATCH] Add command to reload permissions.yml
+
+https://github.com/PaperMC/Paper/issues/49
+
+diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
+index f93ca2e..cd73bfa 100644
+--- a/src/main/java/org/bukkit/Bukkit.java
++++ b/src/main/java/org/bukkit/Bukkit.java
+@@ -1188,4 +1188,11 @@ public final class Bukkit {
+     {
+         return server.spigot();
+     }
++
++    // Paper start
++    public static void reloadPermissions() {
```

Comment:
I don't think this one is needed, it's not going to be a common thing, so lets not litter up the Bukkit util class.

Bukkit.getServer().reloadPermissions() is sufficient API.